include/gridtools/stencil/global_parameter.hpp

```diff
@@ -22,23 +22,26 @@ namespace gridtools {
 
             template <class T>
             struct global_parameter {
-                static_assert(std::is_trivially_copyable<T>(), "global parameter should be trivially copyable");
+                static_assert(std::is_trivially_copy_constructible_v<T>, "global parameter should be trivially copyable");
```

Comment:
Again `trivially_copyable` vs. `trivially_copy_constructible`. If this intended, the comment should probably be adapted, too.